roman_photoz/create_simulated_catalog.py

```diff
@@ -317,21 +315,34 @@ def add_ids(self, catalog):
 
         return new_catalog
 
-    def add_error(self, catalog):
+    def add_error(
+        self, catalog, mag_noise: float = 0.1, mag_err: float = 0.01, seed: int = 42
+    ):
         """
         Add a Gaussian error to each magnitude column in the catalog.
 
+        For each magnitude column, this method adds:
+        - Gaussian noise with a mean equal to the original value and a standard deviation of `mag_noise`.
+        - An error column (`<magnitude_column>_err`) with values sampled from a Gaussian distribution
+          with a mean of 0 and a standard deviation of `mag_err`.
```

Comment:
perhaps this is causing the `Unexpected indentation` error in the RTD build:
```suggestion
        - An error column (`<magnitude_column>_err`) with values sampled from a Gaussian distribution with a mean of 0 and a standard deviation of `mag_err`.
```

Reply:
Thanks for the tips, @zacharyburnett!